theme/dark-green-alt.json

```diff
@@ -0,0 +1,49 @@
+// NOTE: This is a custom JSON file, where lines leading
+// with `//` are removed. No other comments are valid.
+{
+    "foreground": "#eff0f1",
+    "foreground:light": "#ffffff",
+    "background": "#31363b",
+    "background:alternate": "#31363b",
+    "highlight": "#33b833",
+    "highlight:dark": "#2b992b",
+    "highlight:alternate": "#1f991f",
+    "midtone": "#76797c",
+    "midtone:light": "#b0b0b0",
+    "midtone:dark": "#626568",
+    "midtone:hover": "#8a8d8f",
+    "view:checked": "#325c32",
+    "view:hover": "None",
```

Comment:
This should be `"transparent"` and not `"None"`. This is replaced via string replacement directly, so the correct styling would be a passthrough or transparent.